ui/tests/acceptance/config-ui/login-settings-test.js

```diff
@@ -47,11 +47,7 @@ module('Acceptance | Enterprise | config-ui/login-settings', function (hooks) {
   test('it falls back error template if no permission', async function (assert) {
     this.server.get('/sys/config/ui/login/default-auth', () => overrideResponse(403));
     await visit('vault/config-ui/login-settings');
-    assert
```

Comment:
This test change is actually what prompted me to ask you about consistent error handling a few weeks ago! I remember you saying it was on your radar. The new api service error messages display kind of funky. Is the eventual end goal to have them all show similar info? Maybe we don't need to show the URL but that might be useful